pkg/syncer/apiimporter.go

```diff
@@ -189,22 +194,22 @@ func (i *APIImporter) ImportAPIs(ctx context.Context) {
 			clusterctl.GetGVRForLocationInLogicalClusterIndexKey(i.location, i.logicalClusterName, gvr),
 		)
 		if err != nil {
```

Comment:
nit: add the group+resource to the logger in the loop, so each logging call for a particular GR has the data